src/io/platform/f3xx/f302x8/I2Cf302x8.cpp

```diff
@@ -0,0 +1,150 @@
+#include <EVT/io/pin.hpp>
+#include <EVT/io/platform/f3xx/f302x8/I2Cf302x8.hpp>
+
+namespace EVT::core::IO {
+
+/**
+ * Get the I2C ID associated that is associated with the given pin
+ * combination. This information is pulled from the STM32F302x8
+ * documentation with easier documentation on MBed's STM32F302r8
+ * documentation.
+ *
+ * TODO: Make modifications for this to be runnable at compile time
```

Comment:
The old repo has this done for SPI if I remember correctly. Perhaps you could use it as reference.

Reply:
Cool, I'll look to pull that into the future. Right now im focusing on bare minimum functionality, and over the semester working on improvements